JustFakeIt/FakeServer.cs

```diff
@@ -4,13 +4,15 @@
 using System.Linq;
 using Microsoft.Owin.Hosting;
 using Owin;
+using System.Net.Http;
 
 namespace JustFakeIt
 {
     public class FakeServer : IDisposable
     {
         public Uri BaseUri { get; private set; }
         public Expect Expect { get; protected set; }
+        public HttpClient Client { get; private set; }

```

Comment:
Owin.TestServer is one of the ones I was talking of, cause it is in-memory. That being said, I think I was being a bit pedantic, I can see no practical harm in this.